crates/nu-cli/src/shell/completer.rs

```diff
@@ -28,6 +31,17 @@ impl NuCompleter {
             .map(|block| completion::engine::completion_location(line, &block.block, pos))
             .unwrap_or_default();
 
+        let matcher = nu_data::config::config(Tag::unknown())
+            .ok()
+            .and_then(|cfg| cfg.get("completion.style").cloned())
```

Comment:
I don't believe we have any other config.toml keys with a dot in them. I'd prefer to stay consistent so we don't cause confusion. Typicaly we use underscore to separate terms like `completion_style`. Is there a particular reason you wanted to use the dot?

Reply:
OK,

I didnt even think too much about this.

I'll change it to use completion_style.